commons-compiler-tests/src/test/java/org/codehaus/commons/compiler/tests/EvaluatorTest.java

```diff
@@ -368,8 +368,8 @@
         if (comp == "!=") { return lhs != rhs; }
         if (comp == "<")  { return lhs < rhs; }
         if (comp == "<=") { return lhs <= rhs; }
-        if (comp == ">")  { return lhs < rhs; }
-        if (comp == ">=") { return lhs <= rhs; }
+        if (comp == ">")  { return lhs > rhs; }

```

Comment:
This code has been around for a long time  - so wow, this bug is serious o_O! How have all the tests been passing??

Reply:
hehehe, these tests passed because the test code had the same mistake, too: https://github.com/janino-compiler/janino/pull/145/files#diff-18d82b5afcce9ea99d00a37cf2eb08c5992bf20c10856173218cdfab78cb0d4cR398-R399